pages/LearnerDashboard.html

```diff
@@ -0,0 +1,41 @@
+<section id="main_content" class="inner">
+  <h2>
+    <a id="Learner Dashboard" class="anchor"
+       href="#Learner_Dashboard" aria-hidden="true">
+      <span class="octicon octicon-link"></span>
+    </a>
+    Learner Dashboard
+  </h2>
+
+  <p>
+    Oppia's mission from the very start has been - "To help anyone learn anything they want in an effective and enjoyable way.". The "Learner Dashboard" has been designed and developed keeping this goal in mind. If you're a new user and you have signed in for the first time, here's how the learner dashboard would look -
+  </p>
+
+  <img src="static/images/noActivityLearnerDashboard.png" style="width: 100%;">
+
+  <p>
+    The page urges you to go out there and explore, learn new things! Once you start learning the page starts to look a bit different. It's primary goal is to track how much you've learnt which we believe is one of the crucial steps in learning. After all, if you don't know where you are, how will you get to where you want to be?
+  </p>
+
+  <p>
+    Here's how the learner dashboard looks to an experienced learner.
+  </p>
+
+  <img src="static/images/activeLearnerDashboard.png" style="width: 100%;">
```

Comment:
This doesn't really matter unless we want to get _super_ pedantic, but it might be nice to turn those zeroes into larger integers because we're referring to this dashboard as what an experienced learner would see. Definitely not a priority though 😝